docs/griptape-framework/drivers/src/prompt_drivers_3.py

```diff
@@ -1,19 +1,24 @@
 import os
 
+import schema
+
 from griptape.drivers import OpenAiChatPromptDriver
-from griptape.rules import Rule
 from griptape.structures import Agent
 
 agent = Agent(
     prompt_driver=OpenAiChatPromptDriver(
         api_key=os.environ["OPENAI_API_KEY"],
-        temperature=0.1,
-        model="gpt-4o",
-        response_format="json_object",
-        seed=42,
+        model="gpt-4o-2024-08-06",
```

Comment:
Can we include the temperature/seed from past example (if still relevant)?

Reply:
Yeah sure, I added them back